.Lib9c.Tests/Data/TableCSV/Item/EquipmentItemSubRecipeSheetV2.csv

```diff
@@ -0,0 +1,373 @@
+id,required_action_point,required_gold,required_block_index,material_id,material_count,material_2_id,material_2_count,material_3_id,material_3_count,option_id,option_ratio,option_1_required_block_index,option_2_id,option_2_ratio,option_2_required_block_index,option_3_id,option_3_ratio,option_3_required_block_index,option_4_id,option_4_ratio,option_4_required_block_index,min_option_limit,max_option_limit
```

Comment:
I don't know the convention of the sheet in @planetarium/9c-dev team but I wonder simple one. Why `option_id` field doesn't have the number like `2` in `option_2_id` though `option_1_required_block_index` has?

Of course, I know it doesn't affect anything because the field name is just hint and the parser accesses each fields with index. I left this comment for only style.

Reply:
I just guess there are no `option_2_id` and `option_3_id` columns when the `option_id` column added.
Or not, It seems a little habits of writer.
And now we use csv file as state so.. If we want change that yes we can via `PatchTableSheet` action.😉

Reply:
@moreal Can you please review this pull request?